packages/blocks/src/__internal__/utils/types.ts

```diff
@@ -90,10 +90,12 @@ export type ShapeMouseMode = {
 };
 
 export type MouseMode = DefaultMouseMode | ShapeMouseMode;
+export type EdgelessDisplayMode = 'default' | 'grid';
```

Comment:
We should be very cautious to introduce different "modes". Having `EdgelessConfig.showGrid: boolean` or `EdgelessDisplayOptions.showGrid` sounds better to me. In this case, we can define this field as a boolean rather than an enum.